src/screens/Example/AddExampleScreen.js

```diff
@@ -12,6 +12,7 @@ const initialProps = {
 const mapDispatchToProps = (dispatch) => ({
     add: (item) => {
         console.log('add');
```

Comment:
@mtdalton Remove console log

Reply:
This console.log needs to stay here. This is on the ExampleScreen that I use to test. Seeing as this add function is not hooked up to an actual API, I log 'add' so I know that it works. This code isn't used and is not accessible when using the RAT in projects.